src/alert_rules/prometheus/connect.rules

```diff
@@ -34,3 +33,54 @@ groups:
 
         Also look to JVM documentation about threads state:
         https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Thread.State.html
+  # ==============
+  # Connect Tasks Rules
+  # ==============
+  - alert: KafkaConnectTaskFailed
+    expr: kafka_connect_connector_task_status{status="failed"} > 0
+    for: 1m
+    labels:
+      severity: warning
+    annotations:
+      summary: "Kafka Connect task failed"
+      description: "One or more tasks for connector {{ $labels.connector }} have failed."
+  - alert: KafkaConnectConnectorUnassigned
+    expr: kafka_connect_connector_task_status{status="unassigned"} > 0
+    for: 10m
+    labels:
+      severity: warning
+    annotations:
+      summary: "Kafka Connect connector stuck in UNASSIGNED state"
+      description: "Connector {{ $labels.connector }} is not assigned to any worker."
```

Comment:
10 minutes might be too little here, but we should wait to see what actual deployment requirements look like. When using the mirrormaker connectors, if the cluster is idle they go into `UNASSIGNED` if I remember correctly.